### ISSUE TITLE
Missing tablefmt styles in 0.8.10?

### ISSUE THREAD

**Issue by newbiemate** [astanin/python-tabulate#186]
Hi, I installed this package using pip. The version I have is latest (0.8.10).

However some `tablefmt` styles are missing. For example;

        table = [["spam",42],["eggs",451],["bacon",0]]
        headers = ["item", "qty"]
        print(tabulate(table, headers, tablefmt="rounded_outline"))

Will output:

```
item      qty
------  -----
spam       42
eggs      451
bacon       0
```

I noticed that `rounded_grid` is also missing. Are these styles not included in the 0.8.10 pip release?

**Comment by Nyde2283** [comment]
I have the same issue and I use Python 3.10.4 on Ubuntu with tabulate 0.8.10

**Comment by mliszcz** [comment]
I just noticed that as well. The new styles (24ce942b8a588e1152497f321c8e5a9f28bb7b99) were merged just AFTER 0.8.10 release (4892c6e9a79638c7897ccea68b602040da9cc7a7).

@astanin - kindly asking: when do you plan to cut a new 0.8.11 release?

Thanks!

**Comment by stevesimmons** [comment]
@astanin  - Can you please do a new release with the intfmt changes? It's potentially confusing because they are shown on the README but not in the version installed by 'pip install tabulate'.

Thankyou

**Comment by astanin** [comment]
Released in 0.9.0.
